### PR TITLE
feat: Implement private ip

### PR DESCRIPTION
**Pull request by ricantar**
- Implement private IP connection to GCP
- Remove influx from codebase

### CODE REVIEW

**Review by commoddity — COMMENTED** [review]
This looks good to me for now but I will leave to someone else to approve who is more up to date on Relay Meter.

File: .github/workflows/docker-gar.yml [hunk 1]
```diff
@@ -62,8 +62,9 @@ jobs:
       - name: Build the Docker test stack
```
Comment (pablocampogo) [inline .github/workflows/docker-gar.yml]: Also comment the docker stack build, this just takes unnecessary time

File: .github/workflows/lint-test.yml [hunk 2]
```diff
@@ -55,5 +55,6 @@ jobs:
       - name: Build the Docker test stack
         run: docker-compose -f ./testdata/docker-compose.test.yml up -d --build
```
Comment (pablocampogo) [inline .github/workflows/lint-test.yml]: Same comment here

File: e2e_test.go [hunk 3]
```diff
@@ -369,14 +350,12 @@ type (
...
 		TestRelays                  [10]TestRelay
-		influxClient                influxdb2.Client
 		httpClient                  *httpclient.Client
 		startOfDay, endOfDay        time.Time
 		dateParams, orgID, testLBID string
 		options                     TestClientOptions
 	}
 	TestClientOptions struct {
-		db.InfluxDBOptions
 		mainBucket, main1mBucket, orgID,
```
Comment (pablocampogo) [inline e2e_test.go]: U can delete bucket names and org ids of this structure too

**Review by pablocampogo — CHANGES_REQUESTED** [review]
Just some minor comments

**Review by pablocampogo — DISMISSED** [review]

**Review by pablocampogo — APPROVED** [review]

**Pull request merged**